sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetFileFormat.scala

```diff
@@ -127,6 +127,9 @@ class ParquetFileFormat
       conf.setEnum(ParquetOutputFormat.JOB_SUMMARY_LEVEL, JobSummaryLevel.NONE)
     }
 
+    // PARQUET-1580: Disables page-level CRC checksums by default.
```

Comment:
Could you add some comment about the reason why you disable it? It looks like a workaround to avoid Parquet-side performance regression.

Reply:
It will change the data order, please seem https://github.com/apache/spark/pull/26804#discussion_r561044576.

Reply:
Wow. Then, it's a real bug. Thanks for confirmation.